server/src/test/YmlKaoFileVisitor.test.ts

```diff
@@ -325,197 +224,27 @@ describe('Extension Server Tests', () => {
-                            end: {
-                                character: 8,
-                                line: 28,
-                            },
-                            start: {
-                                character: 8,
-                                line: 25,
-                            },
-                        },
-                        uri: '',
-                    },
-                    detail: 'Collection − Text',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Variable,
-                    label: 'collectionWithLevel2',
-                    uri: '',
-                },
-                {
-                    data: 'id_static_functionWithArgsAsBlock',
-                    definitionLocation: {
-                        range: {
-                            end: {
-                                character: 8,
-                                line: 37,
-                            },
-                            start: {
-                                character: 8,
-                                line: 29,
-                            },
-                        },
-                        uri: '',
-                    },
-                    detail: 'Text',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Function,
-                    label: 'functionWithArgsAsBlock',
-                    uri: '',
-                },
-                {
-                    data: 'id_functionWithArgsAsBlock_arg1',
-                    detail: 'Object',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Variable,
-                    label: 'arg1',
-                    scopeEndOffset: 753,
-                    scopeStartOffset: 590,
-                    uri: '',
-                },
-                {
-                    data: 'id_functionWithArgsAsBlock_arg2',
-                    detail: 'Text',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Variable,
-                    label: 'arg2',
-                    scopeEndOffset: 753,
-                    scopeStartOffset: 590,
-                    uri: '',
-                },
-            ];
-            assert.deepEqual(completionProvider.completions, expectedCompletionItems);
+            assert.strictEqual(completionProvider.completions.length, 11);
+            // Three global instances, four local variables.
+            assertHasNelementsRemaining(
+                completionProvider.completions,
+                7,
+                (elem) => elem.kind === CompletionItemKind.Variable,
+            );
+            assertHasNelementsRemaining(
+                completionProvider.completions,
+                4,
+                (elem) => elem.kind === CompletionItemKind.Function,
+            );
             done();
         });
     });
 });
+
+function assertHasNelementsRemaining(
```

Comment:
LOL I was wondering about the weird capitalization (`Nelements` vs. `NElements`). Didn’t realize that was something you wrote.
Does your linter yell at you if you use `…NE…`?

Reply:
I wasn't sure what to use. I think the linter won't yell.